ArduPlane/Log.cpp

```diff
@@ -102,13 +102,19 @@ void Plane::Log_Write_Control_Tuning()
         synthetic_airspeed = logger.quiet_nan();
     }
 
+    int16_t pitch = ahrs.pitch_sensor - g.pitch_trim * 100.0f;
```

Comment:
if we are applying TRIM_PITCH_DEG in fixed wing, then we should also be applying Q_TRIM_PITCH in VTOL modes

Reply:
I could be mistake, but I think Q_TRIM_PITCH is already applies to ahrs.pitch_sensor...maybe its in error or maybe me looking at logs to determine this is wrong...here is an SLT loitering in no wind with a Q_TRIM_PITCH of 3 degrees

Reply:
nope just out of navpitch